rpc/src/rpc_service.rs

```diff
@@ -648,7 +648,7 @@ mod tests {
         )
         .expect("assume successful JsonRpcService start");
         let thread = rpc_service.thread_hdl.thread();
-        assert_eq!(thread.name().unwrap(), "solana-jsonrpc");
+        assert_eq!(thread.name().unwrap(), "solJsonRpcSvc");
```

Comment:
@CriesofCarrots - hey do you remember why we assert the RPC thread name in the tests? I'm just worried the name change might break some loosely coupled code somewhere that is assuming the current name

Reply:
Just kludgy code coverage, afaict. That's been in there since [nearly the dawn of time](https://github.com/solana-labs/solana/pull/1487)!